### DISCUSSION TITLE
Tabs Component not working

### DISCUSSION THREAD

**Discussion by n-drury** [govuk-react/govuk-react#1199] [Q&A] [answered]
Hi, I'm trying to import the tabs component into my project but am getting this error

-    'Tab' is not defined         react/jsx-no-undef

I'm not sure how to resolve this issue

**Comment by penx** [comment]
Can you replicate the problem on codesandbox?

**Reply by n-drury** [reply, marked as answer]
Hi, thanks problem resolved as Tab should be Tabs.Tab in code.
